velox/common/base/SkewedPartitionBalancer.h

```diff
@@ -27,7 +27,7 @@ class SkewedPartitionRebalancerTestHelper;
 /// tasks to busy partition measured by processed data size. This is used by
 /// local partition to scale table writers for now.
 ///
-/// NOTE: this object is not thread-safe.
+/// NOTE: this object is thread-safe.
```

Comment:
Maybe remove the NOTE and just append it at the beginning:
/// This is a thread-safe class used to ...

Because thread safe is not that alertable compared to a thread-unsafe one.

Reply:
Removed. Thanks!